### PR TITLE
Got the "manager" page displaying in WordPress

### PR DESCRIPTION
**Pull request by jameslnewell**
#5 

**Aside:**
I've currently got two apps being built:
- the "manager" app which is the screen displayed when the user clicks the Media menu item
- the "chooser" app which is displayed within the editor

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,13 +19,14 @@ services:
...
     ports:
-      - "8000:80"
+      - '8000:80'
     restart: always
     environment:
       WORDPRESS_DB_HOST: mysql:3306
       WORDPRESS_DB_USER: wordpress
       WORDPRESS_DB_PASSWORD: wordpress
       WORDPRESS_DB_NAME: wordpress
+      WP_MEDIA_CENTER_ENVIRONMENT: development
```
Comment (jameslnewell) [inline docker-compose.yml]: When running in `docker`, we'll force `development` mode.

File: plugin/wp-media-center.php [hunk 2]
```diff
@@ -7,4 +7,23 @@
  * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
  */
 
-echo 'Hello World';
+
+$environment = isset($_ENV['WP_MEDIA_CENTER_ENVIRONMENT']) ? $_ENV['WP_MEDIA_CENTER_ENVIRONMENT'] : 'production';
```
Comment (jameslnewell) [inline plugin/wp-media-center.php]: figure out which environment the plugin is running in

File: plugin/wp-media-center.php [hunk 3]
```diff
@@ -7,4 +7,23 @@
...
+$isDevelopment = $environment === 'development' ? true : false;
+
+function render_manager_page() {
+  echo('<div id="wp-media-center__manager-app"></div>');
+}
+
+function add_manager_scripts() {
+  global $isDevelopment;
+  $url = $isDevelopment ? 'http://localhost:8081/manager.js' : plugin_dir_url(__FILE__).'static/manager.js';
```
Comment (jameslnewell) [inline plugin/wp-media-center.php]: In the `development` environment we use `webpack-dev-server` for `live-reload`. In any other environment we use  assets outputted to the `static` folder.

**Comment by ItsJonQ** [comment]
@jameslnewell Halloo! I tried firing this up locally, but ran into issues 😅 

```
ERROR in ../components/packages/typography/src/body/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/body'
 @ ../components/packages/typography/src/body/index.tsx 15:0-36 35:23-26
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx

ERROR in ../components/packages/typography/src/caption/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/caption'
 @ ../components/packages/typography/src/caption/index.tsx 15:0-36 20:26-29
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx

ERROR in ../components/packages/typography/src/subtitle/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/subtitle'
 @ ../components/packages/typography/src/subtitle/index.tsx 15:0-36 36:27-30
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx

ERROR in ../components/packages/typography/src/title/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/title'
 @ ../components/packages/typography/src/title/index.tsx 15:0-36 39:24-27
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx
```

I have docker running. And the `@salad-ui/components` repo on my computer.
Ran `yarn` for both projects

**Comment by jameslnewell** [comment]
Oh it breaks locally for you?! I can’t reproduce it locally but I’ve been trying to fix it on the fix-ci branch

**Comment by jameslnewell** [comment]
https://github.com/salad-ui/wp-media-center/pull/7

**Comment by ItsJonQ** [comment]
Local is fixed now :). Thanks @jameslnewell !

**Review by ItsJonQ — APPROVED** [review]
🚀 working now!

**Pull request merged**